modules/swagger-codegen/src/main/resources/powershell/Build.ps1.mustache

```diff
@@ -0,0 +1,49 @@
+$ScriptDir = Split-Path $script:MyInvocation.MyCommand.Path
+$ClientPath = ("$ScriptDir\..\..\petstore\csharp\SwaggerClient" | Resolve-Path).ProviderPath
+$PublicPath = "$ScriptDir\src\{{{packageName}}}\Public"
+$BinPath = "$ScriptDir\src\{{{packageName}}}\Bin"
+
+Start-Process -FilePath "$ClientPath\build.bat" -WorkingDirectory $ClientPath -Wait -NoNewWindow
+
+if (!(Test-Path "$ScriptDir\src\{{{packageName}}}\Bin" -PathType Container)) {
+    New-Item "$ScriptDir\src\{{{packageName}}}\Bin" -ItemType Directory > $null
+}
+
+Copy-Item "$ClientPath\bin\*.dll" $BinPath
+
+$Manifest = @{
+    Path = "$ScriptDir\src\{{{packageName}}}\{{{packageName}}}.psd1"
+
+    Author = 'Swagger Codegen Team'
+    CompanyName = 'swagger.io'
+    Description = '{{{packageName}}} - the PowerShell module for {{{appName}}}'
+
+    RootModule = '{{{packageName}}}.psm1'
+    Guid = 'a27b908d-2a20-467f-bc32-af6f3a654ac5' # Has to be static, otherwise each new build will be considered different module
```

Comment:
While this has to be static between module builds, it has to be unique for a given API. I.e, we need deterministic GUIDs. May we could use something like this in the build script: [How can I generate a GUID for a string?](https://stackoverflow.com/a/28608877/4424236).